client/src/components/ProfileCard/ProfileCard.css

```diff
@@ -0,0 +1,37 @@
+.profileCard_card{
+    margin-top: 10px;
+    margin-bottom: 10px;
```

Comment:
I think you can save a line by doing:
`margin: 10px 0;`
The first number will modify the top and bottom, where the second one will modify left and right.

Reply:
This is going to show up multiple times in your code. Not a big deal, but something to keep an eye out on.